cve_bin_tool/util.py

```diff
@@ -82,9 +86,9 @@ def __missing__(self, key):
         return self[key]
 
 
-def regex_find(lines, version_patterns) -> str:
+def regex_find(lines: str, version_patterns) -> str:
     """Search a set of lines to find a match for the given regex"""
-    new_guess = ""
+    new_guess: str = ""
```

Comment:
I don't this this is needed?
```suggestion
    new_guess = ""
```